github/MainClass.py

```diff
@@ -690,6 +692,62 @@ def get_installation(self, id):
         """
         return Installation.Installation(self.__requester, headers={}, attributes={"id": id}, completed=True)
 
+    def create_repo(self, name, description=github.GithubObject.NotSet, homepage=github.GithubObject.NotSet, private=github.GithubObject.NotSet, has_issues=github.GithubObject.NotSet, has_projects=github.GithubObject.NotSet, has_wiki=github.GithubObject.NotSet, is_template=github.GithubObject.NotSet, auto_init=github.GithubObject.NotSet, gitignore_template=github.GithubObject.NotSet, license_template=github.GithubObject.NotSet, allow_squash_merge=github.GithubObject.NotSet, allow_merge_commit=github.GithubObject.NotSet, allow_rebase_merge=github.GithubObject.NotSet):
+        """
+        :calls: `POST /user/repos <http://developer.github.com/v3/repos>`_
+        :param name: string
```

Comment:
Are you able to create repos for any random user, or only yourself? If only for yourself then AuthenticatedUser is the correct place for this method.